doc/user/content/sql/alter-cluster.md

```diff
@@ -116,15 +116,51 @@ displays cluster resource utilization and sizing advice.
 
 #### Resource allocation
 
-The allocation of resources increases proportionally to the size of the cluster.
-For example, a cluster of size `600cc` has 2x as much CPU, memory, and disk as
-a cluster of size `300cc`, and 1.5x as much CPU, memory, and disk as a cluster
-of size `400cc`.
 
 To determine the specific resource allocation for a given cluster size, query
 the [`mz_cluster_replica_sizes`](/sql/system-catalog/mz_catalog/#mz_cluster_replica_sizes)
 system catalog table.
 
+{{< tabs >}}
```

Comment:
We don't have to use tabs. We can instead move the legacy contents into new Appendix pages. For A/B testing, though it'd be easier for people to imagine the content elsewhere whereas the tabbed content might be easier to assess when it's actually rendered.

https://preview.materialize.com/materialize/33899/sql/alter-cluster/#resizing